python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,105 @@
+from utils import create_multi_messages, print_balance_deductions
+
+HOME = os.getenv('HOME')
+NUM_MSGS = int(os.getenv('NUM_MSGS'))
+ 
+logging.basicConfig(format='%(message)s',
+                    level=logging.DEBUG)
+
+parser = argparse.ArgumentParser(description='This program takes inputs for intializing multi message load test.')
+parser.add_argument('-s', '--sender', type = account_type, default = keys_show("account1")[1]['address'], help = 'Sender bech32 address')
+parser.add_argument('-r', '--receiver', type= account_type, default = keys_show("account2")[1]['address'], help= 'Receiver bech32 address')
+parser.add_argument('-n', '--num_txs', type = int, default = 1000, help= 'Number of transactions to be made, atleast should be 1000')
+args = parser.parse_args()
+sender, receiver, NUM_TXS, amount_to_be_sent = args.sender, args.receiver, int(args.num_txs), 1000000
+
+if sender == receiver:
+    sys.exit('Error: The values of arguments "sender" and "receiver" are equal make sure to set different values')
+
+
+#### Fetch balances of sender and receiver accounts before executing the load test ####
+status, sender_balance_old= query_balances(sender)
+if not status:
+    sys.exit(sender_balance_old)
+sender_balance_old = int(sender_balance_old['balances'][0]['amount'])
+
+#### Fetch Balances of receiver executing the load test ####
+status, receiver_balance_old = query_balances(receiver)
+if not status:
+    sys.exit(receiver_balance_old)
+receiver_balance_old = int(receiver_balance_old['balances'][0]['amount'])
+
+### Fetch Sender's expected balance after load test #####
+sender_expected_balance = sender_balance_old - amount_to_be_sent    
+### Fetch Receiver's expected balance after load test###
+
+
+#### Fetching sequence numbers of to and from accounts
+status, seq1_response = query_account(sender)
+if not status:
+    sys.exit(seq1_response)
+
+status, seq2_response = query_account(receiver)
+if not status:
+    sys.exit(seq2_response)
+
+seq1no, seq2no = int(seq1_response['sequence']), int(seq2_response['sequence'])    
+
+
+#### Generating unsigned transactions with a single transfer message 
+status, unsignedTxto = create_unsigned_txs(sender, receiver, amount_to_be_sent, 'unsignedto.json')
+if not status:
+    logging.error(unsignedTxto)
+
+status, unsignedTxfrom = create_unsigned_txs(receiver, sender, amount_to_be_sent, 'unsignedfrom.json')
+if not status:
+    logging.error(unsignedTxfrom)
+        
+for i in range(NUM_TXS):
+        
+#### Duplicating and appending transfer message in the existing array to create a multi-msg transaction        
+    create_multi_messages(NUM_MSGS, 'unsignedto.json')
+    create_multi_messages(NUM_MSGS, 'unsignedfrom.json')
+
+    ### Signing and broadcasting the unsigned transactions from sender to receiver ###
+    seqto = seq1no + i
+    status, txHash = sign_and_broadcast_txs('unsignedto.json', 'signedto.json', sender, seqto)
```

Comment:
do we need to supply `singedto.json` here? I think we can avoid as it's not a dependency of this method
```suggestion
    status, txHash = sign_and_broadcast_txs('unsignedto.json', sender, seqto)
```

Reply:
I think we need to pass it as we have two files for writing signing command output , `signedto.json` and `signedfrom.json`